bar-trender-front-end/src/components/Modals/ModalSelectedElement.js

```diff
@@ -13,12 +13,12 @@ import {
 } from "reactstrap";
 
 // core components
-import image_left from '../assets/img/bg-landing.png';
-import Qr from '../App.js'
+import image_left from '../../assets/img/bg-landing.png';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7163535)
```suggestion
import image_left from "../../assets/img/bg-landing.png";
```